src/WebUI/Components/DaemonSetListingComponent.scss

```diff
@@ -0,0 +1,23 @@
+@import "azure-devops-ui/Core/_platformCommon.scss";
+
+.ds-secondary-text {
```

Comment:
if these are helpful for all other lists, then we should have these in common place